posthog/models/team/team.py

```diff
@@ -153,7 +153,7 @@ class Team(UUIDClassicModel):
     recording_domains: ArrayField = ArrayField(models.CharField(max_length=200, null=True), blank=True, null=True)
 
     primary_dashboard: models.ForeignKey = models.ForeignKey(
-        "posthog.Dashboard", on_delete=models.SET_NULL, null=True, related_name="primary_dashboard_teams"
+        "posthog.Dashboard", on_delete=models.SET_NULL, null=True, related_name="primary_dashboard_teams", blank=True
```

Comment:
This is not a safe change! It breaks types in FE and as a result will cause bugs.

Reply:
As far as I understand it is safe in this case, as `blank` is only used for validating forms. I _definitely_ agree migrations should always be reviewed before merging though.